spec/lib/utils/job-utils/redfish-tool-spec.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node: true */
+
+'use strict';
+
+describe("redfish-tool", function() {
+    var redfishTool;
+    
+    before(function() {
+         helper.setupInjector([
+            helper.require('/lib/utils/job-utils/redfish-tool.js'),
+        ]);
+        redfishTool = helper.injector.get('JobUtils.RedfishTool');
+    });
```

Comment:
What you said is true for test case itself because test cases should always define clear input arguments. But this may impact some test libraries, especially those http request mocking or assertion library, such as *supertest* and *nock*, we heavily rely them on some unit testing. I haven't study those library's implementation, so I am not sure it will use this env variable or not.

The env variable that defined by node.js itself is sensitive and its impact is very broad. It will impact every http request's behavior if it is set by my understanding. I think you don't want to place a line to set this env everywhere to request https, Keep it in Node.js's default setting is convenient. However, if the env is owned by RackHD itself, I think the impact is relative limited.

Anyway, cleanup the spot is always a recommendation from me though there may be no bugs even no cleanup at present. 
Meanwhile, I think this also doesn't bring much code work for your case.